front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -649,13 +654,13 @@ class SettingsCustomizeTab extends PureComponent<
                 onRemove={this.handleHeaderBgOnRemove}
                 errorMessage={headerError}
               />
+              <Outlet
+                id="app.containers.Admin.settings.customize.header_bg_section_field_end"
```

Comment:
Don't we usually give camel-cased names to outlets? Should we be consistent here?

Reply:
Yep, will change!